app/src/main/kotlin/com/wire/android/notification/WireNotificationManager.kt

```diff
@@ -243,6 +245,14 @@ class WireNotificationManager @Inject constructor(
         }
     }
 
+    private suspend fun markConnectionAsNotified(userId: QualifiedID?, connectionRequestUserId: QualifiedID) {
```

Comment:
Maybe `userId` parameter shouldn't be nullable because it's required in order to execute this function properly